src/EnergyPlus/StandardRatings.cc

```diff
@@ -1277,8 +1277,8 @@ namespace StandardRatings {
             }
             // Calculate the standard ratings for multispeed DX cooling coil
             std::map<std::string, Real64> StandardRatingsResult = MultiSpeedDXCoolingCoilStandardRatings(state,
-                                                                                                         DXCoilName,
                                                                                                          DXCoilType,
+                                                                                                         DXCoilName,
```

Comment:
These parameters were flipped causing the routine to fail to recognize cooling coils as it was searching for the string `"Cooling"` in the coil name rather than the coil type.